package.json

```diff
@@ -58,6 +60,7 @@
     "react-loadable": "^5.3.1",
     "react-router-dom": "^4.3.1",
     "redbox-react": "^1.5.0",
+    "slugid": "^2.0.0",
```

Comment:
Don't use this. Use the one from client-web or another web-compatible version.

Reply:
Then you can remove Buffer.

Reply:
Oh that's true! Thanks for pointing that out 😍